RSDKv5/RSDK/Graphics/Drawing.cpp

```diff
@@ -442,6 +442,9 @@ int32 RSDK::GetVideoSetting(int32 id)
             return gameVerInfo.language;
 #endif
         case VIDEOSETTING_CHANGED: return changedVideoSettings;
+#if RETRO_PLATFORM == RETRO_WII
+        case VIDEOSETTING_RUNIN240P: return videoSettings.runIn240p;
+#endif
```

Comment:
While this is a good idea to have a way to set/get this value at runtime (in a future options menu for example), changing it requires reinitializing the video mode on the GX side.
This isn't really needed for now.

Reply:
Definitely!

Reply:
Done!